NaKd.Prerender.io/PrerenderModule.cs

```diff
@@ -146,11 +146,12 @@ private String GetApiUrl(HttpRequest request)
 			}
 
             url = RemoveUtmQueryStrings(url);
-
+            url = RemoveQueryStringByKey(url, "ssr");
             var appendUrl = url.IndexOf("?") >= 0 ? "&ssr=on" : "?ssr=on";
 
             if (ShouldAddMobileParameter(request))
             {
+                url = RemoveQueryStringByKey(url, "mobile");
```

Comment:
Question:
Should we only remove the mobile, if we're supposed to add it, not always?
`ssr` is always removed